File: Python/es_02.py
# 2. Dati in input due lati di un quadrilatero,
# stampare in output se è un quadrato o un rettangolo,
# calcolare l’area e il perimetro.

l1 = int(
    input('Inserisci il primo lato: ')
)

l2 = int(
    input('Inserisci il secondo lato: ')
)

print(
    'È un quadrato.' if l1 == l2 else 'È un rettangolo.'
)

area = l1 * l2
perimetro = 2 * (l1 + l2)

print(f'L\'area è {area}.',
      f'Il perimetro è {perimetro}.', sep='\n'
)

File: Lua/es_02.lua
-- 2. Dati in input due lati di un quadrilatero,
-- stampare in output se è un quadrato o un rettangolo,
-- calcolare l’area e il perimetro.

-- Leggi i lati del quadrilatero
io.write('Inserisci il primo lato (L1): ')
local l1 = io.read('n')

io.write('Inserisci il secondo lato (L2): ')
local l2 = io.read('n')

-- Determina se è un quadrato o un rettangolo
if l1 == l2 then
    print('È un quadrato.')
else
    print('È un rettangolo.')
end

-- Calcola area e perimetro
local area = l1 * l2
local perimetro = 2 * (l1 + l2)

-- Stampa i risultati
print('Area: ' .. area)
print('Perimetro: ' .. perimetro)
